backend/config/config.ts

```diff
@@ -0,0 +1,14 @@
+namespace Config {
+
+    /**
+     * The URL to the MongoDB database
+     * - "mongodb://localhost/conv-tax" for a local MongoDB client
+     * - "mongodb://mongo/conv-tax" for our Docker images
+     * @name Config#MONGO_URL
+     * @type {string}
+     */
+    export const MONGO_URL: string = 'mongodb://localhost/conv-tax';
```

Comment:
Müsste das hier nicht noch geändert werden?

Reply:
Wird es ab jetzt automatisch in der CD (siehe auch Port Umstellung im Frontend). Dadurch ersparen wir uns das manuelle Umstellen.